lib/db_schema_change_warning.rb

```diff
@@ -20,7 +20,7 @@ def migrate(direction) # Executed when a migration runs.
       puts "https://engineering-portal-internal.intellum.com/docs/wikis/ml-data/Requirements-for-tables-ingested-in-our-data-warehouse"
```

Comment:
This link is also broken but I wasn't able to find the replacement, if someone knows it I can fix it too